lib/jerboa/format/body/attribute/xor_mapped_address.ex

```diff
@@ -78,20 +98,20 @@ defmodule Jerboa.Format.Body.Attribute.XORMappedAddress do
     x ^^^ @most_significant_magic_16
   end
 
-  defp ip_4_decode(x) when 32 === bit_size(x) do
+  def ip_4_decode(x) when 32 === bit_size(x) do
```

Comment:
That was not intended, I changed that temporarily for testing. They should be private.